LTTng/CMakeLists.txt

```diff
@@ -21,9 +21,9 @@ if (TRACELOGGING_BUILD_TESTS)
     if (${CMAKE_PROJECT_NAME} STREQUAL ${PROJECT_NAME})
```

Comment:
Github won't let me comment on the ACTUAL line, but on line 3, per discussion: I assume you might want to change this project name to something LTTng specific -- or not, if you want to avoid breaking peoples' command lines.

I suppose it's not very important if you don't intend to connect multiple projects together with a root level cmake.